app/routes/maker.jsx

```diff
@@ -15,12 +15,16 @@ import megadraftStyles from "megadraft/dist/css/megadraft.css";
 // TODO: on save of existing tapesty: make a new function that does a lot of puts.
 
 const fireWebhook = async (url) => {
-  console.log("firing webhook:", url);
-  await fetch(url, {
-    method: "POST",
-  }).then((res) => {
-    console.log(res);
-  });
+  if(url) {
+    console.log("firing webhook:", url);
+    await fetch(url, {
+      method: "POST",
+    }).then((res) => {
+      console.log(res);
+    });
+  } else {
+    console.log("skipping webhook:", url);
+  }
 };
```

Comment:
We should check whether this webhook is necessary at all. From what I understood, this is a Netlify-specific way to rebuild (or restart) the application in order to reload some data. But since we are not using Netlify anymore and we store the data in the database, I think there is no need for such reload.